tracecat/actions/integrations/iam/okta.py

```diff
@@ -44,13 +45,13 @@ def create_okta_client() -> httpx.AsyncClient:
 
 
 @registry.register(
-    default_title="Find Okta User",
-    description="Find an Okta user by login username or email",
+    default_title="Find Okta Users",
+    description="Find Okta users by login username or email",
     display_group="Okta",
     namespace="integrations.okta",
     secrets=[okta_secret],
 )
-async def find_okta_user(
+async def find_okta_users(
```

Comment:
nit: more specific return types help with autogenerated docs, not important atm

Reply:
Got it, I'll go back through my other commits and update so it's clean going forward

Reply:
Just to be clear, are we looking to have defined classes to match the API return types? or is list[dict[str,Any]] what you wanted here?

Reply:
yeah just some form of loose typing like that will do for now, tracking the API is likely too much of a PITA